src/routers/group.ts

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import { GroupService } from '../services/groupService';
+import {
+  groupBodyValidatorOnCreate,
+  groupBodyValidatorOnUpdate,
+  groupRelationsBodyValidator,
+  paramsIdValidator,
+} from '../validation/validators';
+import { Group } from '../models/typeORMModels';
+
+const group = express.Router();
+const groupService = new GroupService();
+
+group.post('/group', groupBodyValidatorOnCreate, async (req, res) => {
+  const group: Group = { ...req.body };
+  const result = await groupService.createGroup(group);
+  if (result instanceof Error) {
```

Comment:
You won't get Error in such a way (errors are thrown, not returned). You will have error handling in next module, I guess, and for now you can leave parts with 2xx and 4xx statuses. Thrown errors get into `next()` function or into catch part of `try...catch`.

Reply:
fixed